sdk/confidentialledger/azure-security-confidentialledger/src/samples/java/com/azure/security/confidentialledger/GetCollectionIds.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public class GetCollectionIds {
+    public static void main(String[] args) {
+        ConfidentialLedgerClient confidentialLedgerClient =
+                new ConfidentialLedgerClientBuilder()
+                        .credential(new DefaultAzureCredentialBuilder().build())
+                        .ledgerUri("https://my-ledger.confidential-ledger.azure.com")
+                        .buildClient();
+        RequestOptions requestOptions = new RequestOptions();
+        Response<BinaryData> response = confidentialLedgerClient.listCollectionsWithResponse(requestOptions);
```

Comment:
I noticed a lot of these samples use the Response overload, is this what we expect customers to use generally? It would be helpful to show what to do with this BinaryData?

I find customers often copying and pasting sample code as-is.

Reply:
I added a more robust example in PostLedger and referenced it in the samples README